### ISSUE TITLE
Timeouts on upload can't be configured 

### ISSUE THREAD

**Issue by sevenseacat** [smeevil/cloudex#71]
Currently the duration for an upload request is limited to hackney's default, which is 5 seconds. This means we're seeing a lot of timeouts for uploading large images (or running a dev server on not-the-world's-best-internet).

This code:
https://github.com/smeevil/cloudex/blob/master/lib/cloudex.ex#L33
looks like it should set it to 60 seconds, but it doesn't. 

The timeout can be manually overridden by adding `timeout` and/or `recv_timeout` options here: 
https://github.com/smeevil/cloudex/blob/master/lib/cloudex/cloudinary_api.ex#L116-L122
But ideally should be configurable from the calling application.

I suspect this will also affect any of the other API operations that have the manual 60-second Task.await call.

**Comment by travelmassive** [comment]
I agree - this would be very helpful. 5 seconds isn't long enough. Hopefully the author can accept the PR :)
